faust/agents/replies.py

```diff
@@ -174,6 +174,7 @@ async def _start_fetcher(self, topic_name: str) -> None:
             self._fetchers[topic_name] = None
             # declare the topic
             topic = self._reply_topic(topic_name)
+            self.app.topics.add(topic)
             await topic.maybe_declare()
             await self.sleep(3.0)
```

Comment:
Not related to your PR directly but thanks for bringing this to my attention- I wonder why `3.0` is a hard-coded value.

Reply:
It works in my test without that `sleep`... might be spurious

Reply:
There's an explicit test for it though :)
```python
       c._drain_replies.assert_called_once_with(topic)
       c.sleep.assert_called_once_with(3.0)
```